pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Description
+
+### Why was this change necessary?
+
+[text here]
+
+### How does it address the problem?
+
+[text here]
+
+### Are there any side effects?
+
+[text here]
+
+### Do you want any specific feedback?
+
+[text here]
+
+### Do you have resources that will help the reviewer? 
+
+[MP-](https://messengerpro.atlassian.net/browse/MP-)
+
+### Zeplin screen design links?
+
+[link here]
+
+### Did you run `./gradlew spotlessApply; ./gradlew testDebugUnitTestCoverage;` locally?
+
+[yes/no]
```

Comment:
Instead we'd have used `Checkboxes` here, right?

Reply:
Yep, I removed this PR template as per our discussion. It would be on website itself not in. base project.